src/selectors/getEntity.test.ts

```diff
@@ -0,0 +1,39 @@
+import { denormalize, schema } from 'normalizr';
+
+import { getEntity } from './getEntity';
+import { ApiDataState } from '../reducer';
+
+test('should return a single entity from normalized data', () => {
+
+    // Set up a fake schema for the getEntity function parameter.
+    const dataSchema = new schema.Entity('users');
+    // const dataListSchema = [dataSchema];
+
+    // Set up apiDataState object.
+    const apiDataState: ApiDataState = {
+        globalConfig: {
+            timeout: 6000
+        },
+        endpointConfig: {
+            getData: {
+                url: 'https://myapi.org/myData',
+                method: 'GET'
+            }
+        },
+        requests: {
+            getData: {
+                networkStatus: 'success',
+                lastCall: 10,
+                duration: 6000,
+                endpointKey: 'getData'
+            }
+        },
+        entities: { users: { abc: '1234' } }
+    };
+
+    // Set up a fake id for the getEntity function parameter.
+    const id = 'abc';
+    const action = getEntity(apiDataState, dataSchema, id);
+
+    expect(action).toEqual(denormalize(id, dataSchema, apiDataState.entities));
+});
```

Comment:
pls add a test for non-existing entities. this should return undefined without an error. 2 cases: schema not found and entity id not found.